packages/vue/RECIPES.md

```diff
@@ -0,0 +1,47 @@
+# Known Issues and recipes - Vue
+
```

Comment:
I would add an information to all framework "recipes" files, that they also should have a look at the "recipes" of the components package, as there are also things described, which apply to all framework wrapper.

Because I have a feeling, that people will otherwise only look for the "recipes" of their frameworks

Reply:
Good suggestion, just added it. I also added a section to the README files of the packages to also reference the known issues files. Please not the link will only work when the branch was merged as we need to use absolute URLs here.